examples/stable/mysql-csi-snapshot/README.md

```diff
@@ -0,0 +1,269 @@
+# Backup and restore MySQL DB using CSI VolumeSnapshot
+
+VolumeSnapshots provide Kubernetes users with a standardized way to copy a volume's contents at a particular point in time without creating an entirely new volume. This functionality enables, for example, database administrators to backup databases before performing edit or delete modifications.
+
+## Introduction
+This document explains how Kanister leverages the use of CSI VolumeSnapshots to take backup and restore of databases in MySQL.
+
+## Prerequisites
+
+- Helm 3 installed
+- Kubernetes 1.16+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+- Kanister controller version 0.71.0 installed in your cluster, let's assume in Namespace `kanister`
+- Kanctl CLI installed (https://docs.kanister.io/tooling.html#install-the-tools)
+- CSI Driver installed
+- VolumeSnapshotClass and StorageClass resources defined (uses above CSI Driver)
+- Snapshot APIs enabled
```

Comment:
I think all these and third point  can be combined into
- VolumeSnapshots should be supported by CSI Driver and respective CRDs should be available